third-party/critical-role/taldorei-campaign-setting/feats.xml

```diff
@@ -0,0 +1,135 @@
+        <sheet>
+            <description>You have advantage on arcana checks when investigating magical objects or devices and can be attuned to a maximum of 4 magical items.</description>
+        </sheet>
+    </element>
+    <element name="Rapid Drinker" type="Feat" source="Tal’Dorei Campaign Setting" id="ID_CRIT_TCS_FEAT_RAPID_DRINKER">
+        <description>
+            <p>Through a lifestyle of hard, competitive drinking, you’ve learned how to quaff drinks at an incredibly accelerated pace. You gain the following benefits:</p>
+            <ul>
+                <li>You can drink a potion as a bonus action, instead of as an action.</li>
+                <li>You have advantage on any saving throws triggered by ingesting an alcoholic or dangerous substance.</li>
+            </ul>
+        </description>
+        <sheet>
+            <description>You can drink potions as a bonus action and have advantage on saving throws against alcoholic or dangerous substances.</description>
+        </sheet>
+    </element>
+    <element name="Spelldriver" type="Feat" source="Tal’Dorei Campaign Setting" id="ID_CRIT_TCS_FEAT_SPELLDRIVER">
+        <description>
+            <p>Through intense focus, training, and dedication, you’ve harnessed the techniques of rapid spellcasting. You are no longer limited to only one non-cantrip spell per turn. However, should you cast two or more spells in a single turn, only one of them can be of 3rd level or higher.</p>
+        </description>
+        <sheet>
+            <description>You are not longer restricted to casting a cantrip as a second spell however your second spell must be of 3rd level or lower.</description>
+        </sheet>
+    </element>
+    <element name="Thrown Arms Master" type="Feat" source="Tal’Dorei Campaign Setting" id="ID_CRIT_TCS_FEAT_THROWN_ARMS_MASTER">
+        <description>
+            <p>You’ve honed your ability to lob weaponry into the fray, including weapons not meant for ranged combat. You gain the following benefits:</p>
+            <ul>
+                <li>Increase your Strength or Dexterity score by 1, to a maximum of 20.</li>
+                <li>Simple and martial melee weapons without the thrown property can be treated as if they have the thrown property. One-handed weapons have a range of 20/60, while two-handed weapons have a range of 15/30.</li>
+                <li>Weapons that naturally have the thrown property increase their range by +20/+40.</li>
+                <li>When you miss with a thrown weapon attack using a light weapon, the weapon immediately boomerangs back into your grasp.</li>
+            </ul>
+        </description>
+        <sheet>
+            <description>Simple and martial melee weapons can be thrown, one handed weapons with a range of 20/60 and two handed weapons with a range of 15/30. Thrown weapons gain additional range of +20/+40 and light thrown weapons that miss bomerang back to you.</description>
+        </sheet>
+        <rules>
```

Comment:
missing ability choice

Reply:
Wdym by ability choice? the feat gives either str or dex so wouldnt it be a choice?

Reply:
yeah, i am blind :bat: